examples/getting-started-app/test_start.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+run_dir='/tmp/cartridge-app-example'
+
+tarantoolctl rocks make
+mkdir -p $run_dir
+
+./../../cartridge start --cfg demo.yml --run_dir $run_dir
+curl -i http://127.0.0.1:8081/admin \
```

Comment:
You need to check example's API for checking your cluster roles. This check is not enough.